### PR TITLE
Js/add in terraform docs

### PR DESCRIPTION
**Pull request by jdsmithit**

### CODE REVIEW

**Review by aleksandra-korczak — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
 # aws-modules-private-ca
 AWS Private CA enables creation of private certificate authority (CA) hierarchies, including root and subordinate CAs
+
+# Documentation
+<!-- BEGIN_TF_DOCS -->
```
Comment (aleksandra-korczak) [inline README.md:5]: Could you please use what they use here:
https://github.com/antonbabenko/pre-commit-terraform#terraform_docs?

I don't think the BEGIN_TF_DOCS works with pre-commit?

**Review by aleksandra-korczak — CHANGES_REQUESTED** [review]
Coul you please use

File: README.md (hunk 1 above)
Comment (jdsmithit) [reply README.md:5]: As we discussed this would mean we don't need pre commit hook

**Review by avnes — APPROVED** [review]

**Review by aleksandra-korczak — APPROVED** [review]

**Pull request merged**